tests/tr1/tests/functional9/test.cpp

```diff
@@ -60,6 +60,21 @@ static void t_invoke() { // test invoke
     CHECK_INT(STD invoke(sum4, 1, 3, 5, 7), 16);
 }
 
+// COMPILE-ONLY

```

Comment:
I wanted to complain about the use of `STD` but now i see it is a tr1 test. 

AFAIK we do not add new Tests to those. Can you have a look whether there is another place in the std Tests that covers reference_wrapper

Reply:
Sure.

Is the place good?

https://github.com/microsoft/STL/blob/main/tests/std/tests/P0357R3_supporting_incomplete_types_in_reference_wrapper/test.compile.pass.cpp

Reply:
I think I will create a new test, because `P0357R3_supporting_incomplete_types` use `usual_20_matrix` and the LWG could use `usual_matrix`